utils/version.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-restricted-properties */
+/* eslint-disable prefer-template */
+/* eslint-disable one-var */
+/* eslint-disable no-underscore-dangle */
+
+'use strict';
+
+const {join} = require('path');
+const chalk = require('chalk');
+const formatDistance = require('date-fns/formatDistance');
+
+const _integrity = Symbol('integrity');
+const _version = Symbol('type');
+const _author = Symbol('author');
+const _created = Symbol('created');
+const _type = Symbol('type');
+const _name = Symbol('name');
+const _org = Symbol('org');
+const _files = Symbol('files');
+const _meta = Symbol('meta');
+
+function readableBytes(bytes) {
+    const i = Math.floor(Math.log(bytes) / Math.log(1024)),
+    sizes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+    return (bytes / Math.pow(1024, i)).toFixed(2) * 1 + ' ' + sizes[i];
+}
+
+class Version {
+    constructor({
+        version = '',
+        integrity = '',
+        author = {},
+        created = null,
+        type = '',
+        name = '',
+        org = '',
+        files = [],
+        meta = [],
+    } = {}) {
+        this.version = version;
+        this.integrity = integrity;
+        this.author = author;
+        this.created = created;
+        this.type = type;
+        this.name = name;
+        this.org = org;
+        this.files = files;
+        this.meta = meta;
+    }
+
+    get version() {
+        return this[_version];
+    }
```

Comment:
Are these getters and setters working? I thought they would collide with same properties on `this`. Does the `this.` values miss a `_`?

Reply:
not sure, will check.

Reply:
I think there is something mixed up with your use of Symbols, this and setters / getters here.

Reply:
Looks like it works I think. In the constructor I set `this.version = version` which calls the setter function which in turn sets the version privately on `this[_version]`. Were you thinking that calling `this.version` in the constructor would override the setter? Or something else I'm missing here?